.github/workflows/pause-community-contributions.yml

```
@@ -16,6 +16,6 @@ permissions:
 jobs:
   pause:
     if: github.repository_owner == 'exercism' # Stops this job from running on forks
-    uses: exercism/github-actions/.github/workflows/community-contributions.yml@community-contributions-reuseable-workflow
+    uses: exercism/github-actions/.github/workflows/community-contributions.yml@3390a06eae3295d427e35adfc15c0bf5d7f32973
```

So this is one exception to our rule of pinning, as we fully control the workflow.
Furthermore, if we pin to a SHA we lose the ability of silently upgrading (which is normally bad, but fine in this case).

```suggestion
    uses: exercism/github-actions/.github/workflows/community-contributions.yml@main
```

---

That makes sense!